scripts/current_validators

```diff
@@ -80,34 +89,48 @@ def parseTransactions(ledger):
 
 def getLedger():
     try:
-        completed = subprocess.run(
-        ['sudo', '/usr/local/bin/read_ledger', '--type', 'pool', '--count'],
-        check=True,
-        stdout=subprocess.PIPE,
-        )
+        if (getpass.getuser() == 'indy'):
+            completed = subprocess.run(
+            ['/usr/local/bin/read_ledger', '--type', 'pool', '--count'],
+            check=True,
+            stdout=subprocess.PIPE,
+            )
+        else:
+            completed = subprocess.run(
+            ['sudo', '/usr/local/bin/read_ledger', '--type', 'pool', '--count'],
+            check=True,
+            stdout=subprocess.PIPE,
+            )
     except subprocess.CalledProcessError as err:
-        logger.error('ERROR attempting to run read_ledger --count subprocess: {}'.format(err))
+        log.error('ERROR attempting to run read_ledger --count subprocess: {}'.format(err))
         raise
     transCount = completed.stdout.decode('utf-8').strip()
-    logging.info("{} entries in ledger.".format(transCount))
+    log.info("{} entries in ledger.".format(transCount))
     try:
-        completed = subprocess.run(
-        ['sudo', '/usr/local/bin/read_ledger', '--type', 'pool', '--frm', '1', '--to', completed.stdout],
-        check=True,
-        stdout=subprocess.PIPE,
-        )
+        if (getpass.getuser() == 'indy'):
+            completed = subprocess.run(
+            ['/usr/local/bin/read_ledger', '--type', 'pool', '--frm', '1', '--to', completed.stdout],
+            check=True,
+            stdout=subprocess.PIPE,
+            )
+        else:
+            completed = subprocess.run(
+            ['sudo', '/usr/local/bin/read_ledger', '--type', 'pool', '--frm', '1', '--to', completed.stdout],
+            check=True,
+            stdout=subprocess.PIPE,
+            )
```

Comment:
May be re-factor this to avoid code duplication?

```python
cmd_args = ['/usr/local/bin/read_ledger', '--type', 'pool', '--frm', '1', '--to', completed.stdout]
if (getpass.getuser() != 'indy'):
        cmd_args = ['sudo'] + cmd_args
completed = subprocess.run(cmd_args, check=True, stdout=subprocess.PIPE)
```